tests/integration/modifiers/did-update-test.js

```diff
@@ -23,5 +23,6 @@ module('Integration | Modifier | did-update', function(hooks) {
     );
 
     this.set('boundValue', 'update');
+    await settled();
```

Comment:
Why is this needed?

Reply:
it's not! I was stabbin' in the dark trying to change the test outcome